libraries/AP_Math/matrix_alg.cpp

```diff
@@ -280,7 +249,7 @@ bool inverse3x3(float m[], float invOut[])
     inv[0] = (m[4] * m[8] - m[7] * m[5]) * invdet;
     inv[1] = (m[2] * m[7] - m[1] * m[8]) * invdet;
     inv[2] = (m[1] * m[5] - m[2] * m[4]) * invdet;
-    inv[3] = (m[5] * m[6] - m[5] * m[8]) * invdet;
+    inv[3] = (m[5] * m[6] - m[3] * m[8]) * invdet;
```

Comment:
this is a rather strange change? Do we have a bug in our existing inverse3x3 code?